app/views/users/_user_card.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="user">
+  <img src=<%= user.photo %> alt="user_photo">
```

Comment:
- [ ] currently your user photo is broken, Kindly fix this
          
![Screenshot from 2022-06-07 16-21-31](https://user-images.githubusercontent.com/66097371/172419355-3cfe0fd3-42c5-4ccb-85c5-593620e20f27.png)

Reply:
@VICTOR-CHUKWU Hello thanks for the review, the image isn't broken, it's just that the link we got from the previous requirements of the project isn't an image link but rather a link to a website with an image.

Reply:
I just tried changing the link to an image that works and it's working correctly.

Reply:
Here is an example of it working:
![image](https://user-images.githubusercontent.com/20567503/172424569-a46c7012-3fed-42e8-8138-af6facd86146.png)

Reply:
@mirouhml  I guess the image link was not pointing to an image, Kindly make a second review request so it can get approved

Reply:
@VICTOR-CHUKWU thanks a lot fo the understanding, I will request another review.